### PR TITLE
Release 3.9.3

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -7,7 +7,7 @@
 %global release 1
 
 Name:    foreman
-Version: 3.9.1
+Version: 3.9.3
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec:10]: Needs npm deps regeneration

File: packages/plugins/rubygem-chunky_png/chunky_png-1.3.10.gem [hunk 2]
```diff
@@ -1 +1 @@
-../.git/annex/objects/W5/j8/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem
\ No newline at end of file
+../../../.git/annex/objects/W5/j8/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem
```
Comment (evgeni) [inline packages/plugins/rubygem-chunky_png/chunky_png-1.3.10.gem:1]: What is this?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-chunky_png/chunky_png-1.3.10.gem (hunk 2 above)
Comment (pcreech) [reply packages/plugins/rubygem-chunky_png/chunky_png-1.3.10.gem:1]: git annex 'magic' that snuck in. no idea.  removing

File: packages/foreman/foreman/foreman.spec (hunk 1 above)
Comment (pcreech) [reply packages/foreman/foreman/foreman.spec:10]: Has been done :D

Comment (evgeni) [reply packages/foreman/foreman/foreman.spec:10]: But with the wrong package.json, this looks like 3.10 or develop

Comment (pcreech) [reply packages/foreman/foreman/foreman.spec:10]: Let's call it a failed cargo-cult copy paste attempt....

`./update-requirements --excludefile ../foreman/package-exclude.json  npm ../foreman/package.json ./packages/foreman/foreman/foreman.spec`

My brain didn't notice the `../foreman/` stuff.  Fixing.

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
Repoclosure can be ignored.

**Pull request merged**